stoplight.js

```diff
@@ -0,0 +1,22 @@
+// Initial Code
+// $("#red").on('click', function() {
+//   $("#red").addClass('active');
+// });
+//
+// $("#yellow").on('click', function() {
+//   $("#yellow").addClass('active');
+// });
+//
+// $("#green").on('click', function() {
+//   $("#green").addClass('active');
+// });
+
+//Refactor
+// $(".bulb").on('click', function() {
+//   $(this).addClass('active');
+// });
+
+//Toggle Functionality
+$(".bulb").on('click', function() {
+  $(this).toggleClass('active');
+});
```

Comment:
i encourage you to keep working on this. think about how you can change the colors of each bulb using existing classes and removing them when another bulb is clicked.